.ci/pipeline_definitions

```diff
@@ -73,7 +74,7 @@ terraformer:
           execute:
           - test-e2e.sh
           depends:
-          - publish
+          - build_oci_image_terraformer
```

Comment:
I suggest you rather change this to:
```
trait_depends:
- publish
```

Reply:
Does it work now? It was not working last time when I was checking it. And I asked you the very same question and back then you didn't replied anything. 😕

Reply:
@ialidzhikov : must have missed your comment back then. I just double-checked in our "cc-demo" repository

Reply:
Thank you for double-checking. It is great that we do not longer need this workaround. 👍